content/en/account_management/governance_console/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Governance Console
+description: Provides a centralized, self-service view of Datadog usage and adoption.
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-governance/"
+  tag: "Blog"
+  text: "Datadog governance 101: From chaos to consistency"
+- link: "https://www.datadoghq.com/blog/volkswagen-organizations/"
+  tag: "Blog"
+  text: "Best practices for managing Datadog organizations at scale"
+- link: "https://docs.datadoghq.com/account_management/audit_trail/"
+  tag: "Documentation"
+  text: "Datadog Audit Trail"
+- link: "https://docs.datadoghq.com/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+---
+
+## Overview
+
+Governance Console unifies the configuration and administration of a large, multi-team, multi-organization Datadog account. For organizations who need centralized governance, strong defaults, and automatic policy enforcement, Governance Console helps establish and maintain a best practice, compliant environment.
+
+Governance Console helps teams understand how their organization is adopting and using Datadog. It provides a centralized, self-service view of Datadog usage and adoption. Governance Console offers control and predictability over the platform's configuration, usage, and cost, helping administrators holistically manage their Datadog deployment. Datadog turns your organization's configuration and usage activity into metrics and insights to guide optimization and investment.
+
+Governance Console supports both new and existing customers, helping new customers start from a well-managed setup, and helping existing customers mature and adopt best practices over time.
```

Comment:
I think this overview section is much too long, a bit repetitive, does too much selling/persuading, and too vague / isn't sufficiently concrete. I think the opening sentence really needs to describe what Governance Console literally is (a page/section in Datadog that displays particular information). We can then offer some brief justification for why you'd want to use it and what specific use-cases it would be most helpful for. I'm also really struggling with the last sentence about how it supports both new and existing customers.... shouldn't all products support new and existing customers?